src/model.js

```diff
@@ -24,29 +32,44 @@ function Model () {}
  * @param {function} callback
  */
 Model.prototype.getData = function (req, callback) {
-  let filename = `${req.params.id}.geojson`
-  fs.readFile(`${dataDir}/${filename}`, (err, dataBuffer) => {
+  const filename = `${req.params.id}.geojson`;
+  const filePath = `${this.dataDirPath}/${filename}`
+  fs.readFile(filePath, (err, dataBuffer) => {
     if (err && err.errno === -2) {
       err.code = 404
-      err.message = `File "${filename}" not found.`
+      err.message = 'File not found'
+      console.log(`${err.message}: ${filePath}`)
```

Comment:
Either remove this or use the Koop logger so that it only prints in debug environments.